client/src/components/traceView.tsx

```diff
@@ -25,15 +25,17 @@ const TraceVisualizer = (data:any) => {
       stylesheet={styleSheet}
       layout={layout}
       style={{
-        width: '70%',
+        width: '100%',
         height: '50rem',
-        border: 'solid',
         objectFit: 'cover',
+        backgroundColor: '#161820'
       }}
+      maxZoom={3}
+      minZoom={0.1}
       cy={cy => {
         myCyRef = cy;
 
-        console.log("EVT", cy);
+        // console.log("EVT", cy);
```

Comment:
Looks like this was removed but want to be sure

Reply:
Yea, I removed the console log just to clean up the console as we're already console logging quite a bit on click to show whats received